chimney.sh

```diff
@@ -28,6 +28,22 @@ export BLOBBER_WALLET_PRIV_KEY=0chainblobwalletprivkey
 
 export DEBIAN_FRONTEND=noninteractive
 
+export PROJECT_ROOT_SSD=/var/0chain/blobber/ssd
+export PROJECT_ROOT_HDD=/var/0chain/blobber/hdd
+
+mkdir -p $PWD/disk-setup/
+wget https://raw.githubusercontent.com/0chain/zcnwebappscripts/enhance/disk-setup/disk-setup/disk_setup.sh -O $PWD/disk-setup/disk_setup.sh
```

Comment:
we might have to move this after line 48 where we do `apt install`

Reply:
done